File: php/i00.subroutine.php
#!/usr/bin/php
<?php
// create subroutine
function  show_date() {
  // WARNING: In php.ini, date.timezone must be set for this to work.
  echo "Today is ",  strftime("%B %d, %Y"), "\n";
}

show_date(); // call subroutine
?>

File: perl/i00.subroutine.pl
#!/usr/bin/env perl -w
use POSIX qw(strftime);  # library to get us strftime

# create subroutine
sub show_date {
  my $date = strftime("%B %d, %Y", gmtime(time));
  print "Today is $date.\n";
}

# call subroutine
show_date
